src/features/modals/CreateToken/CreateToken.module.scss

```diff
@@ -0,0 +1,7 @@
+.CreateToken {
```

Comment:
We generally call these top-level classes `.Container` as they resolve to `[ModuleName]_[Class]` (in this case `CreateToken_CreateToken`).